apps/web-giddh/src/app/gst/gst.component.html

```diff
@@ -1,8 +1,9 @@
 <ng-container appTranslate [file]="'gst'" (localeData)="localeData = $event"
     (commonLocaleData)="commonLocaleData = $event">
     <div class="help-support-aside setting-sidebar-wrapper" *ngIf="asideGstSidebarMenuState === 'in'">
-        <gstr-sidebar (navigateEvent)="handleNavigation($event)" [activeCompanyGstNumber]="activeCompanyGstNumber" [localeData]="localeData" [commonLocaleData]="commonLocaleData"
-            class="gst-sidebar"></gstr-sidebar>
+        <gstr-sidebar (navigateEvent)="handleNavigation($event)" [activeCompanyGstNumber]="activeCompanyGstNumber" (clickOutside)="isMobileScreen ? asideGstSidebarMenuState = 'out' : ''" [exclude]="'#primary-new-header'" [commonLocaleData]="commonLocaleData"
```

Comment:
Don't change the order of input elements

Reply:
reverted